stdlib/Random/src/RNGs.jl

```diff
@@ -291,7 +291,7 @@ function make_seed()
 end
 
 function make_seed(n::Integer)
-    n < 0 && throw(DomainError(n, "`n` must be non-negative."))
+    n &= typemax(n)
```

Comment:
or expand the range?
```suggestion
    n = unsigned(n)
```

Reply:
The simple solutions are often better, yes :facepalm: